tests/test_pool.py

```diff
@@ -64,5 +64,21 @@ def test_list_jobs(self):
         jobs = list(pools[0].jobs())
         self.assertIsNotNone(jobs)
 
+    def test_assign_worker(self):
+        pool_name = 'unit_test_pool_assign_worker_zz'
+        client = FortifySSCClient(self.c.url, self.c.token)
+        self.c.setup_proxy(client)
+        new_pool = client.pools.create(pool_name)
+        pprint(new_pool)
+
+        # Assign a worker that is unassigned to a pool
+        unassigned_worker = [worker['uuid'] for worker in client.workers.list() if worker['cloudPool'] is None]
```

Comment:
this test is predicated on the idea there's an unassigned worker? ergo it's not repeatable. at the least we have to unassign the worker at the start and end of the test so we maintain a constant state

Reply:
great suggestion.  I'll add an assignment to pool 00000000-0000-0000-0000-000000000001 for this test.

Reply:
added test_unassign_worker